core/shared/src/main/scala/spire/std/float.scala

```diff
@@ -6,7 +6,7 @@ import spire.math.Rational
 
 import java.lang.Math
 
-trait FloatIsField extends Field[Float] {
+trait FloatIsField extends Field.WithDefaultGCD[Float] {
```

Comment:
Cool! I had no idea of the laws implemented by the fancy stuff.

Do you remember what the commented out code computed?

If it is approximately lawful, I'll be happy to switch. There are two ways to "prove" that it is approximately lawful.

1) either document that the operation respects lcm(x, y) * gcd(x, y) = x y + commutativity + associativity 

2) or say that the result should match closely the result obtained with Rational; and we can even have exact tests by restricting our Double/Floats to fractions with denominator 2^n